radicle-cli/src/commands/untag.rs

```diff
@@ -4,22 +4,23 @@ use std::str::FromStr;
 use anyhow::anyhow;
 use nonempty::NonEmpty;
 
-use crate::terminal as term;
-use crate::terminal::args::{Args, Error, Help};
 use radicle::cob;
 use radicle::cob::common::Tag;
 use radicle::cob::{issue, patch, store};
 use radicle::crypto::Signer;
 use radicle::storage::{self, WriteStorage};
 
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
 pub const HELP: Help = Help {
     name: "untag",
     description: "Untag an issue or patch",
     version: env!("CARGO_PKG_VERSION"),
     usage: r#"
 Usage
 
-    rad untag <id> <tag>..
+    rad untag <cob-id> <tag> [<option>...]
```

Comment:
```suggestion
    rad untag <cob-id> <tag>... [<option>...]
```

Reply:
since I changed `<id>` to `<cob-id>` here, and there are a few other instances:

```
$ grep -rn "<id>" --include "*.rs" .
./radicle-cli/src/commands/id.rs:29:    rad id rebase <id> [--rev <revision-id>] [<option>...]
./radicle-cli/src/commands/id.rs:30:    rad id show <id> [--rev <revision-id>] [--revisions] [<option>...]
./radicle-cli/src/commands/comment.rs:25:    rad comment <id> [options...]
```

what do you think of changing those as well?